lang/main-nl.json

```diff
@@ -232,7 +232,7 @@
         "screenSharingFailedToInstallTitle": "Installatie van extensie voor het delen van het scherm is mislukt",
         "screenSharingFirefoxPermissionDeniedError": "Er is iets misgegaan tijdens het delen van uw scherm. Controleer of u hier toestemming voor hebt verleend.",
         "screenSharingFirefoxPermissionDeniedTitle": "Scherm delen kan niet worden gestart.",
-        "screenSharingPermissionDeniedError": "Er is iets misgegaan met de machtigingen voor de extensie voor het delen van het scherm. Laad de toepassing opnieuw en probeer het nog eens.",
+        "screenSharingPermissionDeniedError": "Oeps! Er is iets misgegaan met de uitbreidingsrechten voor het delen van het scherm. Laad de toepassing opnieuw en probeer het nog eens.",
```

Comment:
I'd leave the "Oeps!" out.
"permissies" is wellicht  duidelijker dan "uitbreidingsrechten"